doc/whats_new/upcoming_changes/sklearn.neural_network/30712.feature.rst

```diff
@@ -0,0 +1,3 @@
+- Added parameter for `loss` in :class:`neural_network.MLPRegressor` with options
+  `"squared_error"` (default) and `"poisson"` (new).
+  By :user:`Christian Lorentzen <lorentzenchr>`
```

Comment:
What do you think about adding this as an enhancement instead of a feature?

Reply:
New keyword arg and something you couldn’t do before => feature, but I don’t insist.

Reply:
In my opinion both should be fine but if we want to go with feature, I think the title of the PR should also indicate that.